### ISSUE TITLE
Can't build the project

### ISSUE THREAD

**Issue by DamienCassou** [dannyvankooten/browserpass#102]
I'm on Fedora 25. When I want to build the project, I get an error:

```
$ make browserpass-linux64
env GOOS=linux GOARCH=amd64 go build -o browserpass-linux64 ./cmd/browserpass
cmd/browserpass/main.go:8:2: cannot find package "github.com/dannyvankooten/browserpass" in any of:
	/usr/lib/golang/src/github.com/dannyvankooten/browserpass (from $GOROOT)
	($GOPATH not set)
cmd/browserpass/main.go:9:2: cannot find package "github.com/dannyvankooten/browserpass/pass" in any of:
	/usr/lib/golang/src/github.com/dannyvankooten/browserpass/pass (from $GOROOT)
	($GOPATH not set)
makefile:30: recipe for target 'browserpass-linux64' failed
make: *** [browserpass-linux64] Error 1
```

**Comment by DamienCassou** [comment]
A similar issue has already been reported in #100.

**Comment by maximbaz** [comment]
This is probably happening because you don't have $GOPATH defined, and the project is cloned not in $GOPATH. See https://github.com/dannyvankooten/browserpass/issues/100#issuecomment-313599842 on how this has been solved in #100. Or just follow the [build instructions for ArchLinux](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=browserpass), they are pretty much doing the same.

I think ideologically the projects written in `go` must reside in $GOPATH, not just anywhere on the system, that's why all these `cannot find package` issues occur.

I'll close the issue for now.
